.github/workflows/docker.yaml

```diff
@@ -1,13 +1,21 @@
-name: Nebula Service Docker Image Building Workflow
+name: docker
 
 on:
   schedule:
-    - cron:  '0 18 * * *'
+    - cron: '0 18 * * *'
 
 jobs:
-  metad:
-    name: metad
+  docker:
+    name: build docker image
     runs-on: ubuntu-18.04
+    strategy:
+      matrix:
```

Comment:
This will build the same rpm packages five times.Any better way to optimize?

Reply:
Good point. But this is a schedule job, only run one time each day. so I will improve it later.